.devcontainer/devcontainer.json

```diff
@@ -1,4 +1,4 @@
-// cSpell: disable
+// cspell:disable
```

Comment:
While troubleshooting, I copied/pasted the contents of devcontainer.json in the BDD class lab into our devcontainer.json file. It doesn't look like anything broke?